docs/deer/about.md

```diff
@@ -0,0 +1,73 @@
+---
+id: about-deer
+title: Data Entry & Exhibition for Rerum
+custom_edit_url: null
+---
+<!-- A header image is optional; if used should be no greater than 200x600 -->
+![Data Entry & Exhibition for Rerum header image](https://deer.rerum.io/images/deerlogo_banner.jpg)
+
+## Overview
+
+DEER is a framework for designers to quickly template web components for exhibition or data entry. By default, these components all follow the Rerum API, connect to the public sandbox instance of RERUM, and simplify descriptive annotation.
```

Comment:
Can you elaborate on what kind of designers these are? Also, 'exhibition' and 'data entry' are somewhat ambiguous terms.

Reply:
Also, 'descriptive annotation' could be challenging to understand. I think in general, the overview assumes that the reader knows about Rerum and web annotations. It would be better if the overview focused on the problem this project solves/value it brings.